### ISSUE TITLE
A tiny mistake in docs

### ISSUE THREAD

**Issue by SmallSquare** [pytorchbearer/torchbearer#668]
![image](https://user-images.githubusercontent.com/36635227/164865004-7349b63c-7dcf-4fd0-9859-ea45edf91339.png)
Please check the example of Learning Rate Schedulers.
